src/main/java/ladder/domain/Ladder.java

```diff
@@ -0,0 +1,28 @@
+package ladder.domain;
+
+import java.util.List;
+import java.util.stream.IntStream;
+
+public class Ladder {
+    private final RowGenerator rowGenerator;
+    private final List<Row> graph;
+
+    public Ladder(int userLength, Height height, RowGenerator rowGenerator) {
+        this.rowGenerator = rowGenerator;
+        this.graph = initGraph(userLength, height);
+    }
+
+    private List<Row> initGraph(int userLength, Height height) {
+        return IntStream.range(0, height.getHeight())
+                .mapToObj(index -> new Row(userLength, rowGenerator))
+                .toList();
+    }
```

Comment:
```suggestion
    private List<Row> initGraph(int userLength, Height height) {
        List<Row> rows = new ArrayList<>();
        for(int i = 0; i < height.getValue(); i++) {
            Row row = new Row(userLength, rowGenerator);
            rows.add(row);
        }
        return rows;
    }
```

1. Stream의 사용도 좋지만 for문이 가독성이 더 좋을 때도 있습니다! (사용하지 말라는 건 아닙니다)
2. `height.getHeight`도 모호한 표현이군요